syncDev.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 cd ./packages/reason-relay/ && ./build.sh; cd ../../;
+
+cd ./example/ && yarn && cd ../
 rm -rf ./example/node_modules/reason-relay;
```

Comment:
this was missing and so `./example/node_modules` would not exist and the script would fail to copy into node modules etc...